### ISSUE TITLE
PanelChrome: Fix a11y issues of hover header

### ISSUE THREAD

**Issue by polibb** [grafana/grafana#61569]
**What is the problem?**
Cannot reach a hoverable header with a keyboard when going through panels in a dashboard.
 
**To do:**
The hover header should be accessible with a screen reader and a keyboard without hovering the mouse on the panel first.

Problem:

https://user-images.githubusercontent.com/13227501/223991891-527db653-c56d-4e01-9600-db73ccd5a153.mov

**Comment by torkelo** [comment]
@polibb this works for me I can just hit enter, and then arrow key down to move focus around the menu
